ArduPlane/mode_qloiter.cpp

```diff
@@ -29,6 +33,36 @@ void ModeQLoiter::update()
 void ModeQLoiter::run()
 {
     const uint32_t now = AP_HAL::millis();
+
+#if AC_PRECLAND_ENABLED
+    const uint32_t precland_timeout_ms = 250;
```

Comment:
Elsewhere we use a 1 second timeout.

Reply:
my main concern is for users that use QLOITER as bail-out for QLAND if prec-land goes wrong. I wanted it to stop moving towards a bad target as soon as possible
note that there is a race condition with mode change and the lua script. I'd like to find a way to solve this, suggestions welcome